### PR TITLE
Make sure to create a working directory.

### PR DESCRIPTION
**Pull request by clemens-tolboom**
Following the instructions I end up with lot's of files in the current directory.

afaik yo always generate a project directory so let at least instruct people to do so.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,6 +20,12 @@ To install generator-hedley from npm, run:
 npm install -g generator-hedley
 ```
 
+As we generate not only a Web app but also add behat and other scripts make sure to create a project directory.
+
+```bash
+mkdir my-hedley-project
+```
```
Comment (amitaibu) [inline README.md]: maybe worth also to mention `cd my-hedley-project`

**Comment by amitaibu** [comment]
Thanks. Released `v1.3.2`

**Comment by clemens-tolboom** [comment]
:)

**Comment by amitaibu** [comment]
@clemens-tolboom awesome to see you working and contributing to hedley :)

**Pull request merged**